core/src/main/java/org/apache/calcite/model/ModelHandler.java

```diff
@@ -77,15 +84,17 @@ public ModelHandler(CalciteConnection connection, String uri)
     super();
     this.connection = connection;
     this.modelUri = uri;
-    final ObjectMapper mapper = new ObjectMapper();
-    mapper.configure(JsonParser.Feature.ALLOW_UNQUOTED_FIELD_NAMES, true);
-    mapper.configure(JsonParser.Feature.ALLOW_SINGLE_QUOTES, true);
-    mapper.configure(JsonParser.Feature.ALLOW_COMMENTS, true);
+
     JsonRoot root;
+    ObjectMapper mapper;
     if (uri.startsWith("inline:")) {
-      root = mapper.readValue(
-          uri.substring("inline:".length()), JsonRoot.class);
+      String inline = uri.substring("inline:".length()).trim();
```

Comment:
While being a very minor comment, it makes sense to add comments explaining why `trim()` is placed here.

If I read it right, the trim is there to autodetect `inline:            {...}` as json, however that is not that obvious.